util/merkle-mountain-range/src/tests/test_mmr.rs

```diff
@@ -0,0 +1,145 @@
+use super::new_blake2b;
+use crate::{leaf_index_to_pos, util::MemStore, Error, MMRBatch, MerkleElem, Result, MMR};
+use bytes::Bytes;
+use faster_hex::hex_string;
+use lazy_static::lazy_static;
+use proptest::prelude::*;
+use std::convert::TryFrom;
+
+#[derive(Eq, PartialEq, Clone, Debug, Default)]
+struct NumberHash(pub Bytes);
+impl TryFrom<u32> for NumberHash {
```

Comment:
May `u32` => `NumberHash` type conversions fail under some circumstances? I prefer `From` to `TryFrom`.